README.rst

```diff
@@ -203,46 +280,14 @@ The tracing wrapper creates a span for the wrapper handler. This span has the fo
 |                                  | ‘python-lambda-wrapper’          |
 +----------------------------------+----------------------------------+
 
-Sending custom metric from the Lambda function
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code:: python
 
-    import signalfx_lambda
+Steps 7 & 8 don't have (and shouldn't) be done by most users. This is rather info for contributors who want to test changes they do in the wrapper and/or publish a new version. As such, I would vote to have them in last separate section with a title "For contributors" or "For maintainers" or just "Additional info". The metrics and dimensions and tags which are now last are probably of interest to most users, but this two steps are not.
```

Comment:
Looks like copy-paste error, this shouldn't be here, this was a comment.

Reply:
Yes, you are right! ;) I was copying / pasting your comments into the document to help me understand what action to perform. This has been updated.